### DISCUSSION TITLE
Creating queries with system bindings

### DISCUSSION THREAD

**Discussion by GianlucaLostuzzo** [Budibase/budibase#7564] [Help] [answered]
Hi everyone,
I would like to create a query that retrieves from my PostgreSQL database some data and in this query I would like to filter the data retrieve with the current user's email. My query is something like this: 'SELECT name, surname, dob FROM users WHERE email = '{{ Current User.email }}';' but it doesn't return anything and I am sure that there is a row in the table users where the email is equal to the current user's email, so is it possible to use bindings in queries or not?

Thanks.

**Comment by R2bEEaton** [comment, marked as answer]
Hi @GianlucaLostuzzo,

Here's a very similar question that should help #6708.

The short answer is that when you run a query, the client information isn't automatically sent (stuff like Current User). You need to pass the Current User's email to the query using a parameter / binding, which is explained in the discussion above.

Hope this helps! Reply here with any questions or mark this answer as accepted using the check mark above to help future users.

**Comment by GianlucaLostuzzo** [comment]
Thanks for the help! The previous question guided me correctly to the solution.
